### DISCUSSION TITLE
[Question][Hw2]

### DISCUSSION THREAD

**Discussion by Rsln-M** [prosyslab-classroom/cs348-information-security#682] [Q&A]
Name: Ruslan Masimov

Do we have to implement Multiplication in GF(2^4) ourselves? If so, can you give any hints on how to do this? Because I have no idea how to multiply and do modulo polynomials in OCaml.

**Comment by tncks0121** [comment]
There is a multiplication table in the last page, as noted in the phrase "Refer to the Appendix for the table describing the multiplication of two elements in $GF(2^4)$ modulo $x^4 + x + 1$"
